src/main/java/frc/robot/commands/test/DriveFeedForwardCharacterization.java

```diff
@@ -0,0 +1,90 @@
+package frc.robot.commands.test;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.TestConstants;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.FeedForwardCharacterizationData;
+
+/**
+ * Gathers feedforward data for the drivetrain's drive motors.
+ */
+public class DriveFeedForwardCharacterization extends CommandBase {
+
+  private double m_voltage = 0;
+  private FeedForwardCharacterizationData[] m_feedForwardCharacterizationData;
+  
+  private Timer m_timer = new Timer();
+  private Drivetrain m_drive;
+  
+  /**
+   * Creates a new command.
```

Comment:
uh, technically true, but so does every other command. You have a comment for the class which would work for the constructor as you are making that class so for most use cases it would be what it does.